mmhuman3d/models/architectures/mesh_estimator.py

```diff
@@ -431,19 +440,30 @@ def compute_keypoints3d_loss(self, pred_keypoints3d: torch.Tensor,
         pred_keypoints3d = pred_keypoints3d - pred_pelvis[:, None, :]
         loss = self.loss_keypoints3d(
             pred_keypoints3d, gt_keypoints3d, reduction_override='none')
-        valid_pos = keypoints3d_conf > 0
-        if keypoints3d_conf[valid_pos].numel() == 0:
-            return torch.Tensor([0]).type_as(gt_keypoints3d)
-        loss = torch.sum(loss * keypoints3d_conf)
-        loss /= keypoints3d_conf[valid_pos].numel()
+        if has_keypoints3d is None:
```

Comment:
Some comments would be great. i.e. explain under what circumstances will has_keypoints3d be None?